templates/README.template.md

```diff
@@ -2,6 +2,18 @@
 
 One Paragraph of module description goes here
 
+## Connector Attributes (Optional for non-connector extensions)
+
+|  Atrribute  | Value |
+|:-----------:|-------|
+| Bounded     |   No  |
+| Unbounded   |  Yes  |
+| Source      |  Yes  |
+| Sink        |  Yes  |
+| Replayable  |  Yes  |
+| Distributed |   No  |
```

Comment:
a sink can be distributed while a source not, for example elastic-search

Reply:
I see most of the attributes are for sources bounded/unbounded, replayable, distributed